Sanlock installs to /usr/lib64 by default. This is not right on debian

Index: sanlock-3.0.0/src/Makefile
===================================================================
--- sanlock-3.0.0.orig/src/Makefile	2013-07-24 20:56:29.000000000 +0200
+++ sanlock-3.0.0/src/Makefile	2013-07-30 20:32:31.000000000 +0200
@@ -118,7 +118,7 @@
 
 DESTDIR=
 BINDIR=/usr/sbin
-LIBDIR=/usr/lib64
+LIBDIR=/usr/lib
 HEADIR=/usr/include
 MANDIR=/usr/share/man
 
Index: sanlock-3.0.0/wdmd/Makefile
===================================================================
--- sanlock-3.0.0.orig/wdmd/Makefile	2013-07-24 20:56:29.000000000 +0200
+++ sanlock-3.0.0/wdmd/Makefile	2013-07-30 20:32:31.000000000 +0200
@@ -74,7 +74,7 @@
 
 DESTDIR=
 BINDIR=/usr/sbin
-LIBDIR=/usr/lib64
+LIBDIR=/usr/lib
 HEADIR=/usr/include
 MANDIR=/usr/share/man
 
